test/Microsoft.NET.TestFramework/Commands/NuGetRestoreCommand.cs

```diff
@@ -63,7 +63,18 @@ protected override ICommand CreateCommand(params string[] args)
 
             newArgs.AddRange(args);
 
-            return Command.Create(RepoInfo.NuGetExePath, newArgs);
+            if (string.IsNullOrEmpty(TestContext.Current.NuGetExePath))
+            {
+                throw new InvalidOperationException("Path to nuget.exe not set");
+            }
+
+            SdkCommandSpec ret = new SdkCommandSpec()
```

Comment:
nit: var since type is obvious

Reply:
Also, can we name this `spec` instead of `ret`.

Reply:
You have `ret` all over the place. I won't comment on all of them, but I'd prefer readable names.